.github/workflows/maintainer_merge.yml

```diff
@@ -29,7 +29,12 @@ jobs:
       EVENT_NAME: ${{ github.event_name }}
     name: Ping maintainers on Zulip
     runs-on: ubuntu-latest
-    if: github.repository == 'leanprover-community/mathlib4'
+    if: >- # crude check to skip running this on most reviews / comments
```

Comment:
This syntax and whether we can put line breaks here always confused me so I decided to look it up once and for all: [this post](https://stackoverflow.com/a/67532120) (with linked workflow test) summarizes how to add line breaks to conditionals like this. [this post](https://stackoverflow.com/a/21699210) is some useful background on YAML syntax here.

In summary, `>-` combines the following lines and strips all linebreaks, which should make GitHub Actions process this correctly.